source/commandline.md

```diff
@@ -72,14 +72,48 @@ Create a new Meteor project. By default, makes a subdirectory named *name*
 and copies in the template app. You can pass an absolute or relative
 path.
 
-You can use the `--package` option, to create a new package. If used in an
+**Packages**
+
+|| Default | `--bare`  | `--full`  |
+|--------------------|--------|---|---|
+|[autopublish](https://atmospherejs.com/meteor/autopublish)|    X    |   |   |
+|[blaze-html-templates](https://atmospherejs.com/meteor/blaze-html-templates)| X | |X |
+|[ecmascript](https://atmospherejs.com/meteor/ecmascript)|X|X|X|
+|[es5-shim](https://atmospherejs.com/meteor/es5-shim)|X|X|X|
+|[insecure](https://atmospherejs.com/meteor/insecure)|X|||
+|[johanbrook:publication-collector](https://atmospherejs.com/meteor/johanbrook:publication-collector)|||X|
+|[kadira:blaze-layout](https://atmospherejs.com/meteor/kadira:blaze-layout)|||X|
+|[kadira:flow-router](https://atmospherejs.com/meteor/kadira:flow-router)|||X|
+|[less](https://atmospherejs.com/meteor/less)|||X|
+|[meteor-base](https://atmospherejs.com/meteor/meteor-base)|X|X|X|
+|[mobile-experience](https://atmospherejs.com/meteor/mobile-experience)|X|X|X|
+|[mongo](https://atmospherejs.com/meteor/mongo)|X|X|X|
+|[practicalmeteor:mocha](https://atmospherejs.com/meteor/practicalmeteor:mocha)|||X|
+|[reactive-var](https://atmospherejs.com/meteor/reactive-var)|X|X|X|
+|[shell-server](https://atmospherejs.com/meteor/shell-server)|X|X||
+|[standard-minifier-css](https://atmospherejs.com/meteor/standard-minifier-css)|X|X|X|
+|[standard-minifier-js](https://atmospherejs.com/meteor/standard-minifier-js)|X|X|X|
+|[static-html](https://atmospherejs.com/meteor/static-html)||X||
+|[tracker](https://atmospherejs.com/meteor/tracker)|X|X|X|
+
+**Flags**
```

Comment:
@brooksbecton I think we should move the `Flags` section before the `Packages` section. Otherwise we're showing the packages without context - the reason why we're showing this table becomes clear after seeing the `Flags` section.

Reply:
https://github.com/brooksbecton/docs/blob/commandline/add-full-bare-diff/source/commandline.md#meteor-create-name

How does this look?